tests/e2e/rudder-npm-js/src/App.tsx

```diff
@@ -0,0 +1,99 @@
+import logo from './logo.svg';
+import './App.css';
+import { useEffect } from 'react';
+import useRudderStackAnalytics from './useRudderAnalytics';
+import { RudderTyperAnalytics } from './analytics/index';
+
+function App() {
+  const analytics = useRudderStackAnalytics();
+
+  useEffect(() => {
+    if (analytics) {
+      analytics.page('Auto track');
```

Comment:
The app should not contain any kind of raw tracking API calls. Update your tracking plan to include all types of events and update the instrumentation accordingly.

Reply:
done